raider.py

```diff
@@ -0,0 +1,9 @@
+class Raider:
+  def __init__(self, name, ep, gp, in_raid):
```

Comment:
Raider class, 4 properties:
1. name
2. ep
3. gp
4. in_raid a boolean value that when user Login accout_name will set to true, default false